plugins/org.locationtech.udig.libs/.classpath

```diff
@@ -150,6 +145,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/jaxb-api-2.4.0-b180830.0359.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jaxb-runtime-2.4.0-b180830.0438.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/jdom2-2.0.6.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/jfreechart-1.0.13.jar"/>

```

Comment:
looks like this dependency never made it into classpath file but is required for oms:

from `mvn dependency:tree -f pom-libs.xml`

```
[INFO] +- org.jgrasstools:jgt-oms3:jar:0.7.6:compile
[INFO] |  \- jfree:jfreechart:jar:1.0.13:compile
```

however, if it is required, then it should be added to bundle-classpath in MF-File as well